hieradata/site/cp/role/nfsserver.yaml

```diff
@@ -36,6 +36,7 @@ nfs::nfs_exports_global:
       %{facts.networking.ip}/32(ro,nohide,insecure,no_subtree_check,async,root_squash)
       139.229.146.0/24(rw,nohide,insecure,no_subtree_check,async,root_squash)
       139.229.160.0/24(rw,nohide,insecure,no_subtree_check,async,no_root_squash)
+      139.229.165.0/24(rw,nohide,insecure,no_subtree_check,async,no_root_squash)
```

Comment:
Does the entire subnet need to be allowed?  I thought it only needed to be mounted on a single host?

Reply:
i asked Cristian and said the entire 160 segment is OK, but i see your point... will check back on this 👌🏻

Reply:
It is better to allow specific hosts, preferably by fqdn.